### ISSUE TITLE
Feature Request: Suppress Logging/Warning

### ISSUE THREAD

**Issue by nijian84** [websocket-client/websocket-client#865]
When the Websocket-client connects to a WS, it spits out a logging warning. These should be abled to be turned off, by some arg, or off by default, and enabled via the Trace function that exists already. Kinda causes my logs to be noisy.

Thanks!

**Comment by engn33r** [comment]
I cannot tell which warnings you are describing but I expect setting [this to false or removing the `enableTrace` line will help](https://websocket-client.readthedocs.io/en/latest/examples.html#debug-and-logging-options).

**Comment by NadavK** [comment]
I had the same issue with logs being outputted even with Trace set to False:
> 2022-09-18 10:22:01,571 - ERROR - websocket - [Errno -2] Name or service not known - reconnecting
> 2022-09-18 10:22:01,575 - INFO - websocket - reconnect() - retrying in 5 seconds [5 frames in stack]

Fixed with following code:
``` python
        websocket.enableTrace(True, level="FATAL")      # Setting "level" only works if Trace is enabled.
        websocket.enableTrace(False, level="FATAL")     # False still produces disconnect logs
```

**Comment by engn33r** [comment]
This should be solved in v1.5.0 because PR #888 is merged in this release
